package.json

```diff
@@ -8,13 +8,13 @@
   ],
   "dependencies": {
     "@edx/brand": "npm:@openedx/brand-openedx@1.2.2",
-    "@edx/frontend-component-footer": "12.2.0",
+    "@edx/frontend-component-footer": "12.5.2",
     "@edx/frontend-enterprise-catalog-search": "5.5.0",
     "@edx/frontend-enterprise-hotjar": "2.0.0",
     "@edx/frontend-enterprise-logistration": "4.0.5",
     "@edx/frontend-enterprise-utils": "4.0.5",
-    "@edx/frontend-platform": "5.0.0",
-    "@edx/paragon": "20.45.4",
+    "@edx/frontend-platform": "6.2.0",
+    "@edx/paragon": "21.5.6",
```

Comment:
Note: this is not the latest Paragon version as Paragon is now published to a new NPM scope at `@openedx/paragon`. The package `@edx/paragon` has not been published for the last 3 months. Also, v22 is the latest Paragon version at this point (which includes breaking changes for `Chip`, `SearchField`, `Form.Autosuggest` and `Pagination`).

That said, you'll also need to bump `@edx/frontend-platform` to latest v7 to account for peer dependency issues; that is, frontend-platform v7 switches to use the `@openedx` scope for both `frontend-build` and `paragon`.